nf_core/lint/__init__.py

```diff
@@ -205,7 +207,7 @@ def _lint_pipeline(self):
                 "Running lint checks", total=len(self.lint_tests), func_name=self.lint_tests[0]
             )
             for fun_name in self.lint_tests:
-                if self.lint_config.get(fun_name) is False:
+                if self.lint_config.get(fun_name) and self.lint_config.get(fun_name)['run'] is False:
```

Comment:
Minor Python style suggestion (though I think we can drop this anyway, see comment):

```suggestion
                if self.lint_config.get(fun_name, {}).get('run') is False:
```

An optional second argument of `dict.get()` says what it should return if the key isn't here. It defaults to `None`, but if you tell it to return an empty dict instead then you can chain a second `.get()` onto it safely.

Reply:
This is safer than what you've written above, as `self.lint_config[fun_name]` could exist but not be a dict or not have the key `run`. As you're calling `['run']` directly on the value returned by `.get()` (if set) it will fail in both cases.